packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -30,21 +31,61 @@ This guide explains how to customize the sign-in page for both redirect and embe
 
 ## About the sign-in page
 
-The sign-in page is a JavaScript library that gives you a fully-featured and customizable sign-in experience that you can use to authenticate users on any website. How you customize the sign-in page depends on whether Okta is hosting it (redirect authentication) or you are embedding it in your app (embedded authentication).
+The sign-in page is a JavaScript library that gives you a fully featured and customizable sign-in experience that you can use to authenticate users on any website. How you customize the sign-in page depends on whether Okta is hosting it (redirect authentication) or you're embedding it in your app (embedded authentication).
```

Comment:
could try:

How you customize the sign-in page depends if Okta hosts it (redirect authentication) or you embed it in your app (embedded authentication).

Reply:
done